### PR TITLE
add icon extra-animations simple-animations and earth

### PR DESCRIPTION
**Pull request by Eignar17**
add new icons to extra-animations simple-animations your classic icons is not good @soreau  my add icon earth

### CODE REVIEW

**Comment by soreau** [comment]
The new icons look good! I was testing this patch and the ccsm patch when I noticed ccsm doesn't load these 3 new icons if they are in /usr/share/compiz/icons/ but does load them if they are in /usr/share/ccsm/icons/hicolor/scalable/apps/. I am not pushing these for now until I can find out what's going on.

**Comment by Eignar17** [comment]
@soreau  view my patch in to ccsm

**Comment by Eignar17** [comment]
ok view my new code is your create first patch or my

**Comment by Eignar17** [comment]
@soreau  remove classic ccsm :+1:  and reinstall :package:

File: configure.ac [hunk 1]
```diff
@@ -158,9 +158,9 @@ AM_CONDITIONAL(EARTH_PLUGIN, test "x$have_glew" = "xyes" && test "x$have_libcurl
 
 AC_OUTPUT([
 Makefile
-icons/Makefile
```
Comment (soreau) [inline configure.ac]: I don't understand why this change was made. It seems unnecessary.

File: icons/plugin-earth.svg [hunk 2]
```diff
@@ -0,0 +1,538 @@
...
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   version="1.0"
+   width="620"
+   height="620"
+   viewBox="0 0 62 62"
+   id="svg0"
+   sodipodi:version="0.32"
+   inkscape:version="0.46"
+   sodipodi:docname="Ambox_globe.svg"
```
Comment (soreau) [inline icons/plugin-earth.svg]: Can you change this to sodipodi:docname="plugin-earth.svg" please?

**Comment by soreau** [comment]
Closed with 80faeeb, 11390ea and e18bf77. I separated the commits and made the change to plugin-earth.svg for the docname.
